src/components/Send/SendScreen.js

```diff
@@ -750,12 +754,14 @@ class SendScreen extends Component<Props, State> {
             assetsMetadata={availableAssets}
             unselectEnabled={false}
           />
+          {this.state.fee == null &&
+            !!this.state.amount && <ActivityIndicator />}
```

Comment:
design wise, can we make the indicator double the size and have an equal margin to its size from the input above?

Reply:
https://reactnative.dev/docs/activityindicator so let's do large

Reply:
done!